27/09/21  08:41 - HTTPConnectionPool(host='172.70.7.56', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1014207597&pass=1s4B3ll4P&function=update_campaign&campaign_id=WEBCALLP&auto_dial_level=1.0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0c515062e0>: Failed to establish a new connection: [Errno 111] Connection refused')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
27/09/21  08:41 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
27/09/21  08:41 - HTTPConnectionPool(host='172.70.7.56', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1014207597&pass=1s4B3ll4P&function=update_campaign&campaign_id=PORTAPYM&auto_dial_level=1.1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fea02319190>: Failed to establish a new connection: [Errno 111] Connection refused')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
27/09/21  08:41 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
27/09/21  08:41 - HTTPConnectionPool(host='172.70.7.56', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1014207597&pass=1s4B3ll4P&function=update_campaign&campaign_id=MIGRAPYM&auto_dial_level=1.3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9548e970a0>: Failed to establish a new connection: [Errno 111] Connection refused')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 255
27/09/21  08:41 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
